source/applications/dependency-injection.md

```diff
@@ -191,11 +191,17 @@ export default Ember.Component.extend({
 
 ## Factory Instance Lookups
 
-The vast majority of Ember registrations and lookups are performed implicitly.
+You rarely need to write code to look up objects in an Ember application because
```

Comment:
I had written "look up dependencies" at one point, but that seemed pretty abstract. Also `import`ing modules is "looking up dependencies". Maybe "You rarely need to use registered factories directly in an Ember Application"?

Any suggestions here?